src/ResourceManager/Advisor/Commands.Advisor/Cmdlets/AzDisableAzureRmAdvisorRecommendation.cs

```diff
@@ -0,0 +1,163 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
When talking about renaming the files, I meant that they should be renamed to (for instance) DisableAzAdvisorRecommendation.cs.  Right now they all still have the "AzureRm" prefix for the cmdlet within them.  Please update to this format - sorry for not being more clear.